src/test/java/uk/gov/hmcts/reform/civil/sampledata/CaseDataBuilderSpec.java

```diff
@@ -0,0 +1,396 @@
+        return this;
+    }
+
+    public CaseDataBuilderSpec atStateSpec1v2OneDefendantRepresentedUnregisteredOtherUnrepresentedClaimSubmitted() {
+        atStateSpec1v1ClaimSubmitted();
+        addRespondent2 = YES;
+        respondent2 = PartyBuilder.builder().individual().build();
+        respondent1Represented = NO;
+        respondent1OrgRegistered = NO;
+        respondent2Represented = YES;
+        respondent2OrgRegistered = YES;
+        return this;
+    }
+
+    public CaseDataBuilderSpec atStateSpec1v2BothDefendantRepresentedAndUnregistered() {
+        atStateSpec1v1ClaimSubmitted();
+        addRespondent2 = YES;
+        respondent2 = PartyBuilder.builder().individual().build();
+        respondent2SameLegalRepresentative = NO;
+        respondent1Represented = YES;
+        respondent2Represented = YES;
+        respondent1OrgRegistered = NO;
+        respondent2OrgRegistered = NO;
+        return this;
+    }
+
+    public static CaseDataBuilderSpec builder() {
+        return new CaseDataBuilderSpec();
+    }
+
+    public CaseData build() {
+        return CaseData.builder()
+            // Create Claim
+            .legacyCaseReference(legacyCaseReference)
+            .solicitorReferences(solicitorReferences)
+            .claimValue(claimValue)
+            .applicantSolicitor1PbaAccounts(applicantSolicitor1PbaAccounts)
+            .claimFee(claimFee)
+            .applicant1(applicant1)
+            .applicant2(applicant2)
+            .respondent1(respondent1)
+            .respondent2(respondent2)
+            .respondent1Represented(respondent1Represented)
+            .respondent2Represented(respondent2Represented)
+            .respondent1OrgRegistered(respondent1OrgRegistered)
+            .respondent2OrgRegistered(respondent2OrgRegistered)
+            .respondentSolicitor1EmailAddress(respondentSolicitor1EmailAddress)
+            .respondentSolicitor2EmailAddress(respondentSolicitor2EmailAddress)
+            .applicantSolicitor1ClaimStatementOfTruth(applicantSolicitor1ClaimStatementOfTruth)
+            .claimIssuedPaymentDetails(claimIssuedPaymentDetails)
+            .claimFee(claimFee)
+            .applicantSolicitor1CheckEmail(applicantSolicitor1CheckEmail)
+            .applicantSolicitor1UserDetails(applicantSolicitor1UserDetails)
+            .ccdState(ccdState)
+            .ccdCaseReference(ccdCaseReference)
+            .applicant1OrganisationPolicy(applicant1OrganisationPolicy)
+            .respondent1OrganisationPolicy(respondent1OrganisationPolicy)
+            .respondent2OrganisationPolicy(respondent2OrganisationPolicy)
+            .addApplicant2(addApplicant2)
+            .addRespondent2(addRespondent2)
+            .respondent2SameLegalRepresentative(respondent2SameLegalRepresentative)
+            //dates
+            .submittedDate(submittedDate)
+            //ui field
+            .superClaimType(superClaimType == null ? UNSPEC_CLAIM : superClaimType)
```

Comment:
@drummondjm the superClaimType is no longer being set - going forward it is exclusively the CaseAccessCategory

Reply:
updated